app/helpers/github_explorer/api_requests_helper.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+module GithubExplorer
+  module ApiRequestsHelper
```

Comment:
As of now, all of the functionality in this helper module is just to render formatted json for the UI. It doesn't seem very readable to me, so we might want to refactor this into it's own separate class or module and improve the interfaces. It would be cool to add syntax highlighting as well.